net.kvirc.KVIrc.appdata.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018-2020 Flathub maintainers -->
+<component type="desktop">
+  <id>net.kvirc.KVIrc</id>
+  <launchable type="desktop-id">net.kvirc.KVIrc.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>KVIrc</name>
+  <summary>KVIrc is a free portable IRC client</summary>
+  <description>
+    <p>KVIrc is a free portable IRC client based on the excellent Qt GUI toolkit. KVirc is being written by Szymon Stefanek and the KVIrc Development Team with the contribution of many IRC addicted developers around the world.</p>
+  </description>
+  <categories>
+    <category>Communication &amp; News</category>
+  </categories>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://user-images.githubusercontent.com/198756/171932895-b74f5d27-1eaa-4956-9aee-3af191d45b5b.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://www.kvirc.net/</url>
+  <url type="bugtracker">https://github.com/kvirc/kvirc/issues/</url>
+  <url type="help">http://www.kvirc.net/?id=help</url>
+  <developer_name>KVIrc Team</developer_name>
+  <releases>
+    <release version="5.5.0" date="2019-01-01"/>
+  </releases>
+ <content_rating type="oars-1.1">
```

Comment:
Attributes set to `none` can be dropped.

Reply:
Thanks... done